lib/broadside/configuration.rb

```diff
@@ -36,9 +38,36 @@ def ecs
       @ecs ||= EcsConfig.new
     end
 
+    def ssh_cmd(ip, options = { tty: false })
+      cmd = 'ssh -o StrictHostKeyChecking=no'
+      cmd << ' -t -t' if options[:tty]
+      cmd << " -i #{@ssh[:keyfile]}" if @ssh[:keyfile]
+      if @ssh[:proxy]
+        cmd << " -o ProxyCommand=\"ssh #{@ssh[:proxy][:host]} nc #{ip} #{@ssh[:proxy][:port]}\""
+      end
+      cmd << " #{@ssh[:user]}@#{ip}"
+      cmd
+    end
+
     def targets=(_targets)
       raise ArgumentError, ":targets must be a hash" unless _targets.is_a?(Hash)
-      @targets = _targets.map { |name, config| Target.new(name, config) }
+      # transform deploy target configs to target objects
+      @targets = _targets.inject({}) do |h, (target_name, config)|
+        h[target_name] = Target.new(target_name, config)
+        h
+      end
+    end
+
+    def target_from_name!(target_name)
```

Comment:
i also still think it might be a good idea to just make `Broadside.build_deploy(name, options)` and have a method that both finds the `Target` and adds all the `Deploy` options to a `Deploy` object, mostly because i don't think anyone ever is going to want to find a `Target` without building a `Deploy` object out of it.